anybotics_anymal_c/README.md

```diff
@@ -34,6 +34,21 @@ description](https://github.com/ANYbotics/anymal_c_simple_description).
 11. Softened the contacts of the feet to approximate the effect of rubber.
 12. Added `scene.xml` which includes the robot, with a textured groundplane, skybox, and haze.
 
+## MJX
+
+The MuJoCo config in `anymal_c_mjx.xml` has the following edits made specifically for MJX:
+
+* `frictionloss` was removed.
+* PD gain was re-tuned.
```

Comment:
Can you comment why the PD gains had to be re-tuned?

Reply:
I tuned them late 2023, because at the time I found the PD control to be weaker. I re-ran the benchmark just now and the default 100 is fine. I re-ran the notebook and the APG policy learning still works. Removed PD changes!